src/instruction.rs

```diff
@@ -507,7 +620,32 @@ impl crate::Variant for Cmos6502 {
     fn decode(opcode: u8) -> Option<(Instruction, AddressingMode)> {
         // TODO: We obviously need to add the other CMOS instructions here.
```

Comment:
Is this comment still relevant? If so, should we list the missing instructions for future reference here?

Reply:
As far as I can tell the missing instructions are to do with interrupt handling, and so depend on #64 for me to do anything useful here